pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +936,64 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createDataSource(client kubecli.KubevirtClient, name, namespace string) error {
+	labels := make(map[string]string)
+	setDefaultInstancetypeLabels(labels)
+
+	_, err := client.CdiClient().CdiV1beta1().DataSources(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			ds := &v1beta1.DataSource{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      name,
+					Namespace: namespace,
+					Labels:    labels,
+				},
+				Spec: v1beta1.DataSourceSpec{
+					Source: v1beta1.DataSourceSource{
+						PVC: &v1beta1.DataVolumeSourcePVC{
+							Name:      name,
+							Namespace: namespace,
+						},
+					},
+				},
+			}
+			_, err := client.CdiClient().CdiV1beta1().DataSources(namespace).Create(context.Background(), ds, metav1.CreateOptions{})
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("Created new DataSource %s/%s\n", namespace, name)
```

Comment:
Use cmd.Printf

Reply:
It was already mentioned. I replied to the previous discussion thread:
```
Do you want to import a dependency to cmd and change it everywhere, or just this one?
```

Reply:
I can also change it everywhere along when refactoring ugly `setDefaultInstancetypeLabels` that was mentioned by Lee above.